container/shim/start.sh

```diff
@@ -18,6 +18,7 @@ echo "$(date -u) [container] Disk: $(df -h /usr/src/app/shared | awk '(NR>1)')"
 
 # Create if not exists
 mkdir -p /usr/src/app/shared/ssl
+mkdir -p /usr/src/app/shared/nginx
```

Comment:
can we use nginx_conf here? we already have nginx_cache and nginx_log dirs so it would match the convention 

```suggestion
mkdir -p /usr/src/app/shared/nginx_conf
```

Reply:
I was hoping to eventually have everything inside `/nginx`, logs, ssl, config